plugins/utils/sleuth.nix

```diff
@@ -0,0 +1,16 @@
+{
+  config,
+  lib,
+  helpers,
+  pkgs,
+  ...
+}:
+helpers.vim-plugin.mkVimPlugin config {
+  name = "sleuth";
+  originalName = "vim-sleuth";
+  defaultPackage = pkgs.vimPlugins.vim-sleuth;
+
+  maintainers = [lib.maintainers.GaetanLepage];
+
+  # In typical tpope fashion, this plugin has no config options
```

Comment:
That is not true! It at least has the following options:

- `g:sleuth_{filetype}_heuristics`
- `g:sleuth_heuristics`
- `g:sleuth_no_filetype_indent_on`
(I'm not suggesting we package them though, we can do as we want)